src/js/disability-benefits/components/FeaturesWarning.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 export default function FeaturesWarning() {
   return (
     <div className="claims-sidebar-box">
-      <h4 className="claims-sidebar-header">Other claim services</h4>
+      <h4 className="highlight claims-sidebar-header">Other claim services</h4>
```

Comment:
Yeah, I believe that it should be a h2 for the screenreader hierarchy, but the size should be the same size as an h4. 

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/disability-benefits/components/AskVAQuestions.jsx#L7

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/sass/disability-benefits.scss#L609